constants.py

```diff
@@ -0,0 +1 @@
+EXTENSION_TO_SKIP = [".png",".jpg",".jpeg",".gif",".bmp",".svg",".ico",".tif",".tiff"]
```

Comment:
shall we put this in the utils as well? dont want to add an extra file if its not that necessary

Reply:
There so many constant like `generatedDir = "generated"` `openai_model = "gpt-4" ` `openai_model_max_tokens = 20000` which defined over and over in many files. So just though to put it in one place so it will be easier for us to track those constants.